include/gridtools/storage/adapter/nanobind_adapter.hpp

```diff
@@ -97,10 +98,22 @@ namespace gridtools {
                 .template set<property::lower_bounds>(gridtools::array<integral_constant<std::size_t, 0>, ndim>())
                 .template set<property::upper_bounds>(shape);
         }
+
+        template <class T,
+            array_size_t... Sizes,
+            class... Args,
+            class Strides = fully_dynamic_strides<sizeof...(Sizes)>,
+            class StridesKind = sid::unknown_kind>
+        auto as_const_sid(nanobind::ndarray<T, nanobind::shape<Sizes...>, Args...> ndarray,
+            Strides stride_spec_ = {},
```

Comment:
Usually we don’t use trailing `_` in variable names. I see it was done above, so it’s clearly not you who introduced them here, but if you like, you could also fix the names above (`stride_spec_` → `strides_spec` or so).